addons/web_editor/static/lib/jabberwock/jabberwock.css

```diff
@@ -20,6 +20,28 @@ jw-editor {
 jw-toolbar input {
     color: #C6C6C6;
 }
+jw-resizer{
+    display: block;
+    height: 7px;
+    background-color: #eee;
+    border-top: 1px solid #999;
+    text-align: center;
+    width: 100%;
+    line-height: 6px;
+    position: absolute;
+    bottom: 0;
+    cursor: row-resize;
+}
+
+jw-resizer::before {
+    content: "☰";
+    font-size: 6px;
+    margin-top: 1px;
+    display: inline-block;
+    transform: scale(5,1);
+    color: black;
+    vertical-align: middle;
+}
```

Comment:
This entire commit should not be present inside this PR as we disussed in Discord.

Reply:
I thought you said for once we could do it this way. Fine by me...